scripts/mkcloud

```diff
@@ -151,6 +151,7 @@ function show_environment()
     echo "-------------------------------"
     echo " cloudsource: $cloudsource"
     echo "    TESTHEAD: $TESTHEAD"
+    echo "    scenario: $scenario"
```

Comment:
I like the choice of word `scenario` although we should check with @vmoravec that this won't conflict with running of particular `cct` scenarios via `mkcloud`.